src/Domain/Creators/Shell.hpp

```diff
@@ -72,35 +72,33 @@ class Shell : public DomainCreator<3> {
     using type = bool;
     static constexpr Options::String help = {
         "Use equiangular instead of equidistant coordinates."};
-    static constexpr type default_value() noexcept { return true; }
```

Comment:
I would leave this as suggested (as far as I know, (except for debugging/comparison purposes) there is no need to use equidistant coordinates on the Shell domain)